testapi.pm

```diff
@@ -945,24 +952,22 @@ sub script_output {
 
     my $run_script = "/tmp/script$suffix.sh ; echo SCRIPT_FINISHED$suffix-\$?-";
     if (is_serial_terminal) {
-        type_string("/bin/bash -oe pipefail $run_script\n");
+        type_string("/bin/sh -oe pipefail $run_script\n");
```

Comment:
yeah, this still relies on /bin/sh being bash - and as such will fail on debian based systems.

Reply:
IMHO not true because TinyCore provides POSIX sh from busybox and it works there.